mrp_sale_info/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============
+MRP Sale Info
+=============
+
+This module extends the functionality of *sale_mrp*.
```

Comment:
This module extends the functionality of *sale_mrp* and adds related fields to Manufacturing Orders and Work Orders: